geometry_msgs/Point position  	# Desired Cartesian position (x, y, z)
geometry_msgs/Quaternion orientation	# Desired orientation (as quaternion)
geometry_msgs/Twist velocity  	# Desired Cartesian velocity (linear + angular)
